code/controllers/configuration/entries/game_options.dm

```diff
@@ -174,10 +174,22 @@
 /datum/config_entry/string/alert_delta
 	default = "Destruction of the station is imminent. All crew are instructed to obey all instructions given by heads of staff. Any violations of these orders can be punished by death. This is not a drill."
 
-//MONKESTATION EDIT START
+// monkestation start: additional alert levels
 /datum/config_entry/string/alert_epsilon
 	default = "Central Command has ordered the Epsilon security level on the station. Consider all contracts terminated."
-//MONKESTATION EDIT STOP
+
+/datum/config_entry/string/alert_yellow
+	default = "The security alert level has been changed to yellow. At this time, severe structural, atmospheric, or otherwise engineering damage has occurred to %STATION_NAME%. Crewmembers are advised to seek shelter, and Engineering are given enhanced powers for the duration of this alert; including immunity to trespass, breaking and entering, and theft charges committed in service to remedying the danger."
+
+/datum/config_entry/string/alert_amber
+	default = "The security alert level has been changed to amber. There is an ongoing C-B-R-N threat to the station. Crewmembers are advised to don protective gear, and personal oxygen systems until the alert is lifted. Report any symptoms to medical personnel immediately."
+
+/datum/config_entry/string/alert_gamma
+	default = "Central Command has issued an elevated security alert. Security forces are to carry, at the very least, one long-arm and sidearm at all times, and have elevated powers; including termination of non-cooperative Nanotrasen personnel. Civilian personnel are to report to the nearest designated safe-points immediately."
```

Comment:
Can you change the description to be "Central Command has issued an elevated security alert. There is an elevated threat designated by Central Command on board the station. All security forces are to stay armed and on high alert, remaining staff are advised to seek shelter in their departments."

Reply:
To be fair, this does seem pretty trimmed, but we dont want to give sec ALL the power by default. that will lead to potencial issues.

Reply:
Done.